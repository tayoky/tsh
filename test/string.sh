#!/bin/tsh

#string test
echo "string with     spaces"
"echo" "to     files" > "string_file.txt"
"cat" "string_file.txt"
