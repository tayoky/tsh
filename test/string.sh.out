string with     spaces
to     files
